src/sentry/runner/commands/backup.py

```diff
@@ -98,15 +104,22 @@ def import_organizations(src, filter_org_slugs, merge_users, silent):
 
 @import_.command(name="global")
 @click.argument("src", type=click.File("rb"))
+@click.option(
```

Comment:
what will be the default option for self-hosted?

Reply:
The one we have here, since this flag really only benefits self-hosted anyway (we'll never import configs from JSON on SaaS).

The basic reasoning is that, unless they are really super duper sure, in which case they can manually set the flag, users probably want to keep their existing global configs. Things like sentry options and relays change very rarely, so if you are doing a blanket restore on a non-blank DB (if the DB is blank, none of this matters), you probably want the original values.